### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by agununes** [BLavery/virtual-GPIO#6]
is there the possibility of run this with a C program? not python?

thanks

**Comment by BLavery** [comment]
Not really. This project is not current. Rewriting into C is a mega job.

You might consider using a RaspberryPiZero instead of the arduino.
See here:
https://www.raspberrypi.org/blog/gpio-expander/

The RpiZ is emulating a *network* connection to your PC, rather better than
the high speed serial connection of the vgpio project. And the RpiZ is more
"with-it" than an old  '328 arduino.

To drive the GPIO device (the RpiZ) you can use Joan's PIGS functions on
your PC, under any language you want, including C.
http://abyz.me.uk/rpi/pigpio/pigs.html

Good luck
Brian
